packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogLogsPlugin.kt

```diff
@@ -39,6 +39,10 @@ class DatadogLogsPlugin : MethodChannel.MethodCallHandler {
         // See DatadogSdkPlugin's description of this same member
         private var previousConfiguration: Map<String, Any?>? = null
 
+        val instance:DatadogLogsPlugin by lazy {
```

Comment:
this will prevent us from having multiple SDK instances (one of the major feature of SDK v2 arch)?

Reply:
Yes, but unfortunately not supported in Flutter right now and other issues prevent it at the moment anyway. (it wasn't a goal of Flutter v2)

Reply:
Ok, fair. It is just for the future as well, because once we support it in Flutter, we will need to get rid of singleton.